public/app/plugins/datasource/grafana-azure-monitor-datasource/components/ApplicationInsightsEditor/index.tsx

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import { AzureMonitorQuery } from '../../types';
+import { Card, Field, InlineField, InlineFieldRow, Input } from '@grafana/ui';
+
+const ReadOnlyTimeGrain = ({
+  timeGrainCount,
+  timeGrainType,
+  timeGrainUnit,
+}: {
+  timeGrainCount: string;
+  timeGrainType: string;
+  timeGrainUnit: string;
+}) => {
+  const timeFields = timeGrainType === 'specific' ? ['specific', timeGrainCount, timeGrainUnit] : [timeGrainType];
+
+  return (
+    <InlineField label="Timegrain">
+      <>
+        {timeFields.map((timeField) => (
+          <Input value={timeField} disabled={true} onChange={() => {}} key={timeField} width={10} />
+        ))}
+      </>
+    </InlineField>
+  );
+};
+
+const ApplicationInsightsEditor = ({ query }: { query: AzureMonitorQuery }) => {
+  const groupBy = query.appInsights?.dimension || [];
+
+  return (
+    <div data-testid="azure-monitor-application-insights-query-editor">
+      <InlineFieldRow>
+        <InlineField label="Metric" disabled={true}>
+          <Input
+            value={query.appInsights?.metricName}
+            disabled={true}
+            onChange={() => {}}
+            id="azure-monitor-application-insights-metric"
+          />
+        </InlineField>
+        <InlineField label="Aggregation" disabled={true}>
+          <Input value={query.appInsights?.aggregation} disabled={true} onChange={() => {}} />
+        </InlineField>
+      </InlineFieldRow>
+      <InlineFieldRow>
+        <InlineField label="Group by">
+          <>
+            {groupBy.map((dimension) => (
+              <Input value={dimension} disabled={true} onChange={() => {}} key={dimension} />
+            ))}
+          </>
+        </InlineField>
```

Comment:
If there's no groupBy entry then we end up with a weird gap here. Maybe we could have just an empty fallback entry, or just not show the Group by field at all if it's empty?